Workbooks/ADXvsLA.json

```diff
@@ -0,0 +1,1428 @@
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 12
+          },
+          {
+            "id": "363efd16-9c3d-4fca-bd80-b68354883431",
+            "version": "KqlParameterItem/1.0",
+            "name": "fullClusterName",
+            "type": 1,
+            "description": "Cluster name with location",
+            "isRequired": true,
+            "query": "Resources\r\n| where type =~  'Microsoft.Kusto/clusters' and id == '{Resources}'\r\n| project name, id, nameToTrim = tostring(properties.dataIngestionUri), location\r\n| project new = strcat(name,'.',location)",
+            "crossComponentResources": [
+              "{Subscription}"
+            ],
+            "isHiddenWhenLocked": true,
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "97b31e4f-86e7-4b55-b982-33020a053e14",
+            "version": "KqlParameterItem/1.0",
+            "name": "ClusterName",
+            "type": 1,
+            "description": "Only the clustername ",
+            "query": "Resources\r\n| where type =~  'Microsoft.Kusto/clusters' and id == '{Resources}'\r\n| project name",
+            "isHiddenWhenLocked": true,
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "4b566ef9-09ac-4cab-8e54-70395109cc9e",
+            "version": "KqlParameterItem/1.0",
+            "name": "justDB",
+            "label": "parsed DB name",
+            "type": 1,
+            "description": "The database name",
+            "query": "project filterName = '{selectADXcluster:label}'\r\n| summarize by filterName\r\n| extend DBname = split(filterName,\"/\")[1]\r\n//| parse filterName with * '/Databases/' DBname '\"' *\r\n| project DBname",
+            "crossComponentResources": [
+              "{Subscription}"
+            ],
+            "isHiddenWhenLocked": true,
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "c64f6a0c-e581-4890-8330-34f6f3b7b82a",
+            "version": "KqlParameterItem/1.0",
+            "name": "Workspace",
+            "type": 5,
+            "description": "Azure Sentinel workspace from which the data is duplicated",
+            "query": "where type =~ 'microsoft.operationalinsights/workspaces'\r\n| project id",
+            "value": "/subscriptions/1c61ccbf-70b3-45a3-a1fb-848ce46d70a6/resourceGroups/Admin/providers/Microsoft.OperationalInsights/workspaces/AdminSOC",
```

Comment:
Same here... Subscription and Resource details should not be mentioned. Please update.
Validations are failing because of this.

Reply:
resolved